docs/building.md

```diff
@@ -84,6 +85,27 @@ sudo apt install cmake ninja-build python3 \
                  llvm
 ```
 
+## Prerequisites to build on macOS
+
+These steps were tested on macOS Catalina (10.15).
+
+First, you will need to install dependencies:
+
+- Install Python 3.x
+- Install [Xcode](https://developer.apple.com/xcode/download/)
+   - You also need to install the `XCode Command Line Tools` by running 
+   `xcode-select --install`. Alternatively, if you already have the full Xcode 
+   installed, you can find them under the menu 
+   `Xcode -> Open Developer Tool -> More Developer Tools...`. This step will 
+   install `clang`, `clang++`, and `make`.
+- Install [Homebrew](https://brew.sh/)
+- Install `ninja` and `cmake` running the following command:
+```
+brew install ninja cmake
+```
+
+# Build instructions:
```

Comment:
Shouldn't this have an extra "#".  This is part of the UNIX-like section?

Reply:
Yes, you right. I just fixed it.